### PR TITLE
feat(natives/vehicle): Update SetConvertibleRoof

### PR DESCRIPTION
**Pull request by ahcenezdh**
Here is a more detailed documentation for SetConvertibleRoof.

### CODE REVIEW

**Comment by ahcenezdh** [comment]
Can i change the argument p1 and put something like roofState instead?

**Comment by 4mmonium** [comment]
> Can i change the argument p1 and put something like roofState instead?

Yes you can, maybe you can name it `toggle` as well?

File: VEHICLE/SetConvertibleRoof.md [hunk 1]
```diff
@@ -7,9 +7,39 @@ ns: VEHICLE
 // 0xF39C4F538B5124C2 0xC87B6A51
 void SET_CONVERTIBLE_ROOF(Vehicle vehicle, BOOL p1);
 ```
-
+# This allows for the vehicle's roof to be put on when set to true, and removed when set to false, provided that the vehicle has a version with a roof and a version without a roof.
```
Comment (4mmonium) [inline VEHICLE/SetConvertibleRoof.md:10]: Remove the `#`.

File: VEHICLE/SetConvertibleRoof.md [hunk 2]
```diff
@@ -7,9 +7,39 @@ ns: VEHICLE
...
 
 ## Parameters
-* **vehicle**: 
-* **p1**: 
+* **vehicle**: The vehicle to which the roof state will be applied.
+* **p1**: Boolean value where `true` adds the roof and `false` removes it, applicable only if the vehicle has a version with and without a roof.
+
+## Vehicles with both roofed and roofless versions (others may exist; this list is compiled from decompiled scripts).
```
Comment (4mmonium) [inline VEHICLE/SetConvertibleRoof.md:16]: No need for the double hash (`##`), maybe you can do this:

```
#### Vehicles with both roofed and roofless versions (others may exist; this list is compiled from decompiled scripts).

* Chino
* voltic
* buccaneer
* buccaneer2
* chino2
* faction
* faction2
* mamba
```

File: VEHICLE/SetConvertibleRoof.md [hunk 3]
```diff
@@ -7,9 +7,39 @@ ns: VEHICLE
...
 void SET_CONVERTIBLE_ROOF(Vehicle vehicle, BOOL p1);
 ```
-
+# This allows for the vehicle's roof to be put on when set to true, and removed when set to false, provided that the vehicle has a version with a roof and a version without a roof.
 
 ## Parameters
-* **vehicle**: 
-* **p1**: 
+* **vehicle**: The vehicle to which the roof state will be applied.
+* **p1**: Boolean value where `true` adds the roof and `false` removes it, applicable only if the vehicle has a version with and without a roof.
```
Comment (4mmonium) [inline VEHICLE/SetConvertibleRoof.md:14]: Maybe rename `p1` to `toggle`, or `roofState`, but I believe this toggles a roof on/off.

File: VEHICLE/SetConvertibleRoof.md [hunk 4]
```diff
@@ -7,9 +7,39 @@ ns: VEHICLE
 // 0xF39C4F538B5124C2 0xC87B6A51
 void SET_CONVERTIBLE_ROOF(Vehicle vehicle, BOOL p1);
```
Comment (4mmonium) [inline VEHICLE/SetConvertibleRoof.md:8]: Maybe rename `p1` to `toggle`, or `roofState`, but I believe this toggles a roof on/off.

**Review by 4mmonium — CHANGES_REQUESTED** [review]
Hey there, couple changes needed, good otherwise! Thank you for your contribution 👍

**Comment by ahcenezdh** [comment]
Should be good @4mmonium !

File: VEHICLE/SetConvertibleRoof.md [hunk 5]
```diff
@@ -5,11 +5,41 @@ ns: VEHICLE
...
+#### Vehicles with both roofed and roofless versions (others may exist; this list is compiled from decompiled scripts).
+
+* chino
+* voltic
+* buccaneer
+* buccaneer2
+* chino2
+* faction
+* faction2
+* mamba
```
Comment (4mmonium) [inline VEHICLE/SetConvertibleRoof.md:25]: All this needs to be moved above the `## Parameters` section (under 'This allows for...'), thank you.

File: VEHICLE/SetConvertibleRoof.md [hunk 6]
```diff
@@ -5,11 +5,41 @@ ns: VEHICLE
...
+
+## Examples
+```lua
+-- In this case we are removing the roof from the vehicle.
+local vehicle = GetVehiclePedIsIn(PlayerPedId(), false)
+SetConvertibleRoof(vehicle, false)
+```
+
+```javascript
```
Comment (4mmonium) [inline VEHICLE/SetConvertibleRoof.md:34]: Whoops, I must have missed this, replace with `` ```js ``, not javascript, thanks!

Comment (ahcenezdh) [reply VEHICLE/SetConvertibleRoof.md]: No problem, should be good

File: VEHICLE/SetConvertibleRoof.md (hunk 5 above)
Comment (4mmonium) [reply VEHICLE/SetConvertibleRoof.md:25]: You missed this in your last commit  :-)

Comment (ahcenezdh) [reply VEHICLE/SetConvertibleRoof.md:21]: Oh sorry, it's good like this?

**Pull request merged**